src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -67,106 +67,118 @@
+    {{ redeterminationDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.CCJ_REQUESTED %}
-      {{ ccjRequestedForDefendantDashboard(claim) }}
+    {{ ccjRequestedForDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.CLAIMANT_ACCEPTED_ADMISSION_AND_REQUESTED_CCJ or status === ClaimStatus.CLAIMANT_ALTERNATIVE_PLAN_WITH_CCJ %}
-      {{ admissionByCCJDefendantDashboard(claim) }}
+    {{ admissionByCCJDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.CLAIMANT_ACCEPTED_ADMISSION or status === ClaimStatus.CLAIMANT_ACCEPTED_COURT_PLAN_SETTLEMENT
       or status === ClaimStatus.CLAIMANT_ACCEPTED_ADMISSION_AND_DEFENDANT_NOT_SIGNED %}
-      {{ settlementAgreementSignedByClaimantDefendantDashboard(claim) }}
+    {{ settlementAgreementSignedByClaimantDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.ELIGIBLE_FOR_CCJ_AFTER_FULL_ADMIT_PAY_IMMEDIATELY_PAST_DEADLINE %}
-      {{ claimFullAdmissionForDefendantDashboard(claim) }}
+    {{ claimFullAdmissionForDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.ADMISSION_SETTLEMENT_AGREEMENT_REACHED %}
-      {{ settlementAgreementSignedByBothDefendantDashboard(claim) }}
+    {{ settlementAgreementSignedByBothDefendantDashboard(claim) }}
     {% elif status === ClaimStatus.RESPONSE_SUBMITTED %}
-      {% if claim.response.responseType === domain.ResponseType.FULL_DEFENCE %}
-        {% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
-          {{ claimRejectedAlreadyPaidForDefendantDashboard(claim) }}
+    {% if claim.response.responseType === domain.ResponseType.FULL_DEFENCE %}
+      {% if claim.response.defenceType === DefenceType.ALREADY_PAID %}
+        {{ claimRejectedAlreadyPaidForDefendantDashboard(claim) }}
         {% elif claim.response.freeMediation === FreeMediationOption.NO %}
-          {{ claimRejectedFreeMediationNOForDefendantDashboard() }}
-        {% else %}
-          {{ claimRejectedFreeMediationYESForDefendantDashboard() }}
-        {% endif %}
+        {{ claimRejectedFreeMediationNOForDefendantDashboard() }}
+      {% else %}
+        {{ claimRejectedFreeMediationYESForDefendantDashboard() }}
+      {% endif %}
       {% elif claim.response.responseType === domain.ResponseType.PART_ADMISSION %}
-        {% if claim.response.paymentDeclaration %}
-          {{ claimPartAdmittedAlreadyPaidForDefendantDashboard(claim) }}
-        {% else %}
-          {{ claimPartAdmittedToOweForDefendantDashboard(claim) }}
-        {% endif %}
-      {% elif claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
-        {{ claimFullAdmissionForDefendantDashboard(claim) }}
+      {% if claim.response.paymentDeclaration %}
+        {{ claimPartAdmittedAlreadyPaidForDefendantDashboard(claim) }}
+      {% else %}
+        {{ claimPartAdmittedToOweForDefendantDashboard(claim) }}
       {% endif %}
+      {% elif claim.response.responseType === domain.ResponseType.FULL_ADMISSION %}
```

Comment:
I'm not sure about
```
{% endif %}
{% elif ... %}
```
Can you double-check these `elif` branches can be accessed?

Reply:
Yes they are accessed